.github/workflows/tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - name: Install Ruby gems with Bundler
       run: bundle install
     - name: Lint podspec using local source
-      run: pod lib lint --verbose --allow-warnings
```

Comment:
Why remove this? Were we allowing warnings for using deprecated methods before?

Reply:
Yes, this was in place to accommodate warnings that are being address with this PR.

Reply:
Got it. 👍